services.html

```diff
@@ -0,0 +1,84 @@
+          <h3>UX / UI</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
+        </div>
+        <div class='service-box'>
+          <h3>Digital Marketing</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
+        </div>
+        <div class='service-box'>
+          <h3>Web Development</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
+        </div>
+        <div class='service-box'>
+          <h3>iOS Development</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
+        </div>
+        <div class='service-box'>
+          <h3>Android Development</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
+        </div>
+      </section>
+      <div class='line'></div>
+      <section class='bottom-content'>
+        <div class='service-facts'>
+          <h3>Some Facts About Our Services</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+          <ul>
+            <li>Awesome thing</li>
+            <li>Amazing thing</li>
+            <li>Cool thing</li>
+            <li>Great thing</li>
+          </ul>
+        </div><img alt="one of our employees hard at work"
+                 class="services-info-img"
+                 src="img/services-info.png">
```

Comment:
The idea of separating out a HTML tag's properties is great for readability. Consider doing it like this to make it a little more uniform and easier to read:
```
<img
    alt="one of our employees hard at work"
    class="services-info-img"
    src="img/services-info.png"
/>
```